packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -124,7 +124,8 @@ const PostsPagePostHeader = ({post, classes}: {
   const { major } = extractVersionsFromSemver(post.version)
   const hasMajorRevision = major > 1
   const wordCount = post.contents?.wordCount || 0
-  
+  const readTime = post.readTimeMinutes ?? 1
```

Comment:
Do we want to show "0 min read" if `readTimeMinutes` is 0? I feel like sticking with a minimum of 1 is clearer

Reply:
Good point! I made the change directly in the resolver, so this should also fix the problem of entering negative numbers as we'll just return 1 instead.